### PR TITLE
scan move

### PR DESCRIPTION
**Pull request by parvdesai-tenable**
Depends on vm/scans module #52

### CODE REVIEW

File: tenable/utilities/scan_move/__init__.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            if count == limit:
+                break
+
+            if scan_instance.get('status', '') == 'completed':
+                count += 1
+                scan_history.append(scan_instance.get('id'))
+
+        return scan_history
+
+    def move(self, limit):
```
Comment (harshshah-tenable) [inline tenable/utilities/scan_move/__init__.py:42]: Please add docstring for the methods and update the PR description as well

**Review by harshshah-tenable — APPROVED** [review]

File: tenable/utilities/scan_move/__init__.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+.. rst-class:: hide-signature
+.. autoclass:: ScanMove
+    :members:
+'''
+from tenable.io import TenableIO
+
+
+class ScanMove:
+    '''
+    This will contain all methods related to Users
```
Comment (Mangusingh-te) [inline tenable/utilities/scan_move/__init__.py:19]: Is docstring for above method correct ?

File: tenable/utilities/scan_move/__init__.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    '''
+
+    def __init__(self, source_tio: TenableIO, target_tio: TenableIO):
+        self.source_tio = source_tio
+        self.target_tio = target_tio
+
+    def _get_scan_history(self, scan_id, limit):
```
Comment (Mangusingh-te) [inline tenable/utilities/scan_move/__init__.py]: Specify type hint for method input params along with return values.

File: tenable/utilities/scan_move/__init__.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        scan_history = []
```
Comment (Mangusingh-te) [inline tenable/utilities/scan_move/__init__.py:28]: Add docstring for the method.

File: tenable/utilities/scan_move/__init__.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+            ]
+        }
+
+        for mv_scan in self.source_tio.v3.vm.scans.search(
+            filter=scan_filter
+        ):
+            scan_history = self._get_scan_history(mv_scan['id'], limit)
+
+            for scan in scan_history:
+                print("Exporting Scan ID:{}, with history_id: {} now\n".format(
```
Comment (Mangusingh-te) [inline tenable/utilities/scan_move/__init__.py]: Use **logger** instead of **print**.

File: tenable/utilities/scan_move/__init__.py [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+                    mv_scan, scan
+                ))
+
+                scan_report = self.source_tio.v3.scans.export(scan)
+                imported_scan = self.target_tio.v3.scans.import_scan(
+                    scan_report
+                )
+
+                print("Scan imported: {}".format(
```
Comment (Mangusingh-te) [inline tenable/utilities/scan_move/__init__.py]: Use logger instead of **print**.

**Review by Mangusingh-te — CHANGES_REQUESTED** [review]
Added minor comments.

File: tenable/utilities/scan_move/__init__.py (hunk 1 above)
Comment (bhavin-te) [reply tenable/utilities/scan_move/__init__.py:42]: I have addressed.

File: tenable/utilities/scan_move/__init__.py (hunk 3 above)
Comment (bhavin-te) [reply tenable/utilities/scan_move/__init__.py]: I have addressed.

File: tenable/utilities/scan_move/__init__.py (hunk 4 above)
Comment (bhavin-te) [reply tenable/utilities/scan_move/__init__.py:28]: I have addressed.

File: tenable/utilities/scan_move/__init__.py (hunk 5 above)
Comment (bhavin-te) [reply tenable/utilities/scan_move/__init__.py]: I have seen everywhere print is used so let have consistency.

File: tenable/utilities/scan_move/__init__.py (hunk 6 above)
Comment (bhavin-te) [reply tenable/utilities/scan_move/__init__.py]: I have seen everywhere print is used so let have consistency.

**Review by alpesh-te — APPROVED** [review]
LGTM! need to fixes some minor what that can be handle with integration testing

**Pull request merged**